extensions/tls-registry/cli/src/main/java/io/quarkus/tls/cli/TlsCommand.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.tls.cli;
+
+import java.util.concurrent.Callable;
+
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "tls-registry", sortOptions = false, header = "Install and Manage TLS development certificates", subcommands = {
```

Comment:
After the use of the `tls` alias, this should be renamed to `tls` too.
I am wondering, if its possible to infer the alias from the root command name to further simplify things. Captured some thoughts in: #41656

Reply:
I've set it to `tls`.